### DISCUSSION TITLE
Amasty Countdown integreated

### DISCUSSION THREAD

**Discussion by ppeyman** [breezefront/community#326] [3. Integrations]
Hi dears i Need to Integreated Amasty countdown 
But i don't know how to done it can someone help me ? i tried many ways but i can't complete

**Comment by vovayatsyuk** [comment]
Have you tried following this [guide](https://breezefront.com/docs/integration-example)?
Did you find component names (entry points)?

**Reply by ppeyman** [reply]
i added js file but i have problem with below section 

![image](https://github.com/user-attachments/assets/91661332-3c2b-4cc3-82ff-1f8f7634dae2)


It is confusing for me .

**Comment by vovayatsyuk** [comment]
I've just added an integration with this module: https://github.com/breezefront/module-breeze-amasty-promo-countdown

**Reply by ppeyman** [reply]
Thank You !!
